VideoAPI/Testing.Common/Configuration/ZAPConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Testing.Common.Configuration
+{
+    public class ZAPConfiguration
+    {
+        public string ApiAddress { get; set; }
+        public int ApiPort { get; set; }
+        public string Apikey { get; set; }
+        public string WorkingDirectory { get; set; }
+        public int MinutesToWait { get; set; }
+        public bool RunZap { get; set; }
```

Comment:
This sounds like it should be CanRunZap ???

Reply:
Just to denote run Zap